### ISSUE TITLE
Halcyon.Sample.Ini Economy Section is Missing

### ISSUE THREAD

**Issue by Vinhold** [InWorldz/halcyon#70]
The released sample Halcyon.ini files does not have the Economy section in it. This is the only place where the price values can be set for controlling all uploads, group creation and other settings related to fees and economy controls. While most grids wont need or use money operations this is still the only option open to control access to upload and group creation in a grid if those features are to be restricted to only approved persons. 

It looks like this was the only place to set the default prim limit for any region, though this might only be a redundant entry since the value is set in each region.

[Economy]

```
; 45000 is the highest value that the sim could possibly report because of protocol constraints
ObjectCapacity = 30000
; Money Unit fee to upload textures, animations etc
PriceUpload = 0
; Money Unit fee to create groups
PriceGroupCreate = 0
; This is the account Money goes to for fees.  Remember, economy requires that money circulates somewhere... even if it's an upload fee
;EconomyBaseAccount = <UUID of Account that holds the starting cash>
```

**Comment by kf6kjg** [comment]
Any comment improvements you can suggest? Jims got a PR is for this now: #444

**Comment by appurist** [comment]
As I mentioned in #444, I think many of the entries above (like the UserLevel, Stipend-related, etc) are old options copied from OpenSim but not used in Halcyon. I think some of the comments are very specific to OpenSim concepts that don't apply here, and I don't think we should be attempting to document any other options in the sample INI file unless they are confirmed to still apply and work correctly in Halcyon. I think the ObjectCapacity option should not be used from INI file, and may be ignored in some parts of the code in favor of the expected region.xml file option.

I have added an `[Economy]` section in PR #444, and with the key entries including an unused `EconomyBaseAccount` and comment, along with `PriceUpload` and `PriceGroupCreate`.  That's a good start. 

I think additional entries will require specific source-level investigation to see if they are still appropriate and/or even supported at all.  Given the many higher priorities, I don't think this will happen any time soon, but we could leave this open on the backlog since we really should identify the options supported and document those.

**Comment by Vinhold** [comment]
My point was to have the [Economy] section restored and what items in it that are being used. Anything else is nice if it is known what it does and be better documented if it is supported.

**Comment by appurist** [comment]
Okay that's great, and should be resolved by #444.  And as we discover or make use of any other options that have code behind them, we can add those to the Halcyon.sample.ini as well.

**Comment by kf6kjg** [comment]
#444 has been merged for some time. Closing.
